src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,29 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ goods, currentTab, set }) => {
+  return (
+    <div className="tabs is-boxed">
+      <ul>
+        {goods.map(good => (
+          <li
+            data-cy="Tab"
+            key={good.id}
+            className={cn`${good.id === currentTab.id && 'is-active'}`}
```

Comment:
The `classnames` library usage seems incorrect. The `cn` function should be called with parentheses, not template literals. It should be `cn({ 'is-active': good.id === currentTab.id })`.